app/forms/declaration/InlandModeOfTransportCode.scala

```diff
@@ -23,7 +23,21 @@ import play.api.data.{Form, Forms}
 import play.api.libs.json.Json
 import utils.validators.forms.FieldValidator.isContainedIn
 
-case class InlandModeOfTransportCode(inlandModeOfTransportCode: Option[String] = None)
+case class InlandModeOfTransportCode(inlandModeOfTransportCode: Option[String] = None) {
+  import TransportCodes._
+
+  def extractModeValue: Option[String] = inlandModeOfTransportCode.map {
```

Comment:
Enummmmm

Reply:
Yeah, it was old implementation, I just added possibility to retrieve text based on the code.

Reply:
Again - not the right way to get display text

Reply:
Similar, moved to use messages